auto-generated/cluster/stone-prd-rh01/tenants/insights-core-tenant/appstudio.redhat.com_v1beta2_integrationtestscenario_insights-puptoo-tekton-testing.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: appstudio.redhat.com/v1beta2
+kind: IntegrationTestScenario
+metadata:
+  labels:
+    test.appstudio.openshift.io/optional: "true"
+  name: insights-puptoo-tekton-testing
+  namespace: insights-rules-tenant
```

Comment:
there is a mismatch with the namespaces now. This says `insights-rules-tenant` but is inside the `insights-core-tenant` folder

Reply:
Updated.